data/filedefs/filetypes.systemverilog

```diff
@@ -0,0 +1,62 @@
+# For complete documentation of this file, please see Geany's main documentation
```

Comment:
I don't think SystemVerilog should be implemented as a builtin filetype. Have a look at e.g. `filetypes.JSON.conf` how to add a non-builtin filetype. Under the `[settings]` section, add `tag_parser=SystemVerilog` and `lexer_filetype=verilog`.

Rename this file to `filetypes.Systemverilog.conf` so it becomes a non-builtin filetype

Reply:
Oookay, that makes sense.  That's how I initially thought this was meant to be done, but eventually I mistakenly understood that the only way to add SystemVerilog parsing was as a separate builtin language.  Overall, I had a hard time figuring out what needed to be added and what would be reused.

So in summary, internally there are file types (lexer, GEANY_FILETYPES, highlighting, etc) and tag managers (ctags, TM_PARSER, etc).  I should nix the former and keep the latter, correct?  I thought they were somewhat related.

So the only changes in C/C++ code I get to keep are:
1. Adding SystemVerilogParser
2. Adding the stuff I added to tm_parser